spyder/plugins/variableexplorer/widgets/collectionseditor.py

```diff
@@ -66,6 +66,8 @@
             DataFrameEditor)
 
 
+MAC = sys.platform == 'darwin'

```

Comment:
Please remove this constant. We use `sys.platform == 'darwin'` wherever is needed instead.